addons/flannel/0.20.2/yaml/kube-flannel.yml

```diff
@@ -82,7 +82,7 @@ data:
     }
   net-conf.json: |
     {
-      "Network": "10.244.0.0/16",
+      "Network": "10.32.0.0/20",
```

Comment:
was this not changed in the base?

also how will this work with existing flannel installations?

Reply:
I'll revert this, because it shouldn't have been included

but this line isn't actually used (it's overridden by a patch)